src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -33,24 +33,28 @@ export const coreVitals = (): void => {
 		name: string;
 		value: number;
 	};
+	
+	const nineDigitPrecision = (value: number) => {
+		// The math functions are to make sure the length of number is <= 9
+		return Math.round(value * 1_000_000) / 1_000_000;
+	}
```

Comment:
I was getting some prettier errors that were preventing me from pushing. I found out it was because you are missing `;` here.

It might be worth checking prettier is working on your IDE (I have it run on file save).

Reply:
(full disclosure: no IDE was used for this PR)